packages/components/src/components/tree/component.tsx

```diff
@@ -136,16 +136,53 @@ export class KolTreeWc implements TreeAPI {
 				event.preventDefault();
 				break;
 			}
+			case 'Right':
 			case 'ArrowRight': {
-				await currentTreeItem.expand();
-				event.preventDefault();
+				if (!(await currentTreeItem.isOpen())) {
+					await currentTreeItem.expand();
+					event.preventDefault();
+				} else {
+					await openItems[currentIndex + 1]?.focusLink();
+					event.preventDefault();
+				}
 				break;
 			}
+			case 'Left':
 			case 'ArrowLeft': {
-				await currentTreeItem.collapse();
+				if (await currentTreeItem.isOpen()) {
+					await currentTreeItem.collapse();
+					event.preventDefault();
```

Comment:
`preventDefault` brauchen wir in jedem Fall, ist dupliziert -> Wir können es vor dem `if`-Block platzieren/